config/locales/services/fr.yml

```diff
@@ -2,111 +2,111 @@
       providers:
         apple:
           name: "%{first} %{last}"
-        not_available: Provider %{name} is not available!
-        not_enabled: Provider %{name} is not enabled!
+        not_available: Le fournisseur %{name} n'est pas disponible !
+        not_enabled: Le fournisseur %{name} n'est pas activé !
     badges:
       award_beloved_comment:
-        message: "You're famous! [This is the comment](%{comment}) for which you're being recognized. 😄"
+        message: "Vous êtes célèbre ! [C'est le commentaire](%{comment}) pour lequel vous êtes reconnu. 😄"
       award_streak:
-        longest: 16 weeks! You've achieved the longest writing streak possible. This makes you eligible for special quests in the future. Keep up the amazing contributions to our community!
+        longest: 16 semaines ! Vous avez atteint la plus longue série d'écriture possible. Cela vous rend éligible pour des quêtes spéciales à l'avenir. Continuez à apporter votre contribution à notre communauté !
         message:
-          one: "Congrats on achieving this streak! Consistent writing is hard. The next streak badge you can get is the %{count} Week Badge. 😉"
-          other: "Congrats on achieving this streak! Consistent writing is hard. The next streak badge you can get is the %{count} Week Badge. 😉"
+          one: "Félicitations pour avoir atteint cet objectif ! Il est difficile d'écrire de façon constante. Le prochain badge que vous pouvez obtenir est le badge %{count} Semaine. 😉"
+          other: "Félicitations pour avoir atteint cet objectif ! Il est difficile d'écrire de façon constante. Le prochain badge que vous pouvez obtenir est le badge %{count} Semaine. 😉"
       award_yearly_club:
         message:
-          one: Happy %{community} birthday! Can you believe it's been %{count} year already?!
-          other: Happy %{community} birthday! Can you believe it's been %{count} years already?!
-      congrats: Congrats!!!
-      thank_you: Thank you so much for your contributions!
+          one: Joyeux %{community} anniversaire ! Peux-tu croire que ça fait déjà %{count} an ?!
+          other: Joyeux %{community} anniversaire ! Peux-tu croire que ça fait déjà %{count} ans ?!
+      congrats: Félicitations !
+      thank_you: Merci beaucoup pour vos contributions !
     broadcasts:
       welcome_notification:
         generator:
-          welcome: 'Welcome Notification: %{key}'
+          welcome: 'Notification de bienvenue : %{key}'
     error_messages:
       clean:
-        parse_error: There was a problem parsing the front-matter YAML. Perhaps you need to escape a quote or a colon or something. Email %{email} if you are having trouble.
+        parse_error: Il y a eu un problème pour analyser la matière première YAML. Vous devez peut-être échapper un guillemet, un deux-points ou autre chose. Envoyez un courriel à %{email} si vous avez des problèmes.
```

Comment:
Since the surrounding context is a yaml parse error, actual chevrons `«` would have been ignored as other text and not treated as punctuation.

https://fr.wikipedia.org/wiki/Guillemet#Guillemets_dactylographiques

I think I answered my own question, this is not something that would confuse a user, and _guillemet_ here includes single and double quotes.